hazelcast/src/main/java/com/hazelcast/map/impl/mapstore/MapDataStore.java

```diff
@@ -32,7 +32,7 @@
 
     V add(K key, V value, long now);
 
-    void addTransient(K key, long now);
+    V addTransient(K key, V value, long now);
```

Comment:
Can you add some javadoc to this method? What is the purpose of the returned value?